snap/snapcraft.yaml

```diff
@@ -99,54 +57,90 @@ parts:
     override-prime: |
       echo 'Skip Prime'
   fractal:
-    after: [rustup, gtksourceview, ffmpeg]
-    parse-info: [usr/share/metainfo/org.gnome.Fractal.metainfo.xml]
+    after: [ rustup, libshumate, pipewire ]
+    parse-info: [ usr/share/metainfo/org.gnome.Fractal.metainfo.xml ]
     source: https://gitlab.gnome.org/GNOME/fractal.git
     source-type: git
-    source-tag: $SNAPCRAFT_PROJECT_VERSION
+    source-tag: '5.rc1'
     plugin: meson
-    meson-parameters: [--prefix=/usr, -Dprofile=default]
+    meson-parameters: [--prefix=/snap/fractal/current/usr, -Dprofile=default, -Doptimization=3, -Dbuildtype=release]
     build-packages:
       - libssl-dev
       - libgstreamer1.0-dev
       - libgstreamer-plugins-bad1.0-dev
       - libgstreamer-plugins-base1.0-dev
       - libgstreamer-plugins-good1.0-dev
-      - libges-1.0-dev
-      - libhandy-0.0-dev
-    stage-packages:
-      - libgspell-1-2
-      - gstreamer1.0-gtk3
-      - gstreamer1.0-plugins-bad
-      - gstreamer1.0-plugins-base
-      - gstreamer1.0-plugins-good
-      - gstreamer1.0-plugins-ugly
-      - libges-1.0-0
-      - libgstreamer1.0-0
-      - libgstreamer-plugins-bad1.0-0
-      - libgstreamer-plugins-base1.0-0
-      - libgstreamer-plugins-good1.0-0
-      - libhandy-0.0-0
+      - libpipewire-0.3-dev
```

Comment:
you can remove this, you are already building it

Reply:
Ahh.. sorry, missed it